lib/linguist/heuristics.yml

```diff
@@ -395,12 +394,6 @@ disambiguations:
     named_pattern: key_equals_value
   - language: Java properties
     pattern: '^[^#!][^:]*:'
-- extensions: ['.props']
-  rules:
-  - language: XML
-    pattern: '^(\s*)(?i:<Project|<Import|<Property|<\?xml|xmlns)'
-  - language: INI
-    pattern: '(?i:\w+\s*=\s*)'
```

Comment:
Why were these removed (`Text` in `.builds` and all of `.props`)? How is this related to Solidity?

Reply:
These were actually heuristics that didn't have a file extension listed in `languages.yml`. So in order to implement the generic-extension feature correctly, I had to remove a few things.

Reply:
Ah, I see. Makes sense :slightly_smiling_face: 

My main concern is that we will lose the context on this change with the squash (future contributors will have a hard time finding this discussion buried in the PR). Could you maybe send it as a separate PR and I'll be happy to validate it before end of day?

Reply:
> future contributors will have a hard time finding this discussion buried in the PR

That reminds me, I should rename this PR and clarify that the generic-extension feature is the primary change, with Solidity support being the impetus.

Reply:
Alright, done. @pchaigno, anything else?